mv3/addon/rules.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "id": 1,
+    "priority": 1,
+    "action": {
+      "type": "redirect",
+      "redirect": {
+        "transform": { "scheme": "http", "host": "localhost", "port": "8080" }
+      }
+    },
+    "condition": { "urlFilter": "||ipfs.io", "resourceTypes": ["main_frame"] }
+  },
+  {
+    "id": 2,
+    "priority": 1,
+    "action": {
+      "type": "redirect",
+      "redirect": {
+        "transform": { "scheme": "http", "host": "localhost", "port": "8080" }
```

Comment:
:thought_balloon: given that port will be something that can be changed by the user, we will most likely end up adding these as dynamic rules during startup.